README-cmake.md

```diff
@@ -88,6 +88,19 @@ I (0) cpu_start: Starting scheduler on APP CPU.
 Hello, world!
 ```
 
+## Simulate
+
+Wokwi for VS Code allows to simulte our application:
+0. `diagram.json` migth need to be updated with the proper board. See, [available boards](https://docs.wokwi.com/diagram-format#parts).
+1. Press `F1`
+2. Run `Wokwi: Start Simulator`
+
+For more information and details on how to use the Wokwi extension, see [Getting Started] and [Debugging your code] Chapter of the Wokwi documentation.
+
+[Getting Started]: https://docs.wokwi.com/vscode/getting-started
```

Comment:
Nit: maybe just use embedded links inside the paragraph, rather than listing these separately?